index.js

```diff
@@ -28,6 +28,8 @@ const productionEndpoints = {
     kip17: 'https://kip17-api.klaytnapi.com',
     kip7: 'https://kip7-api.klaytnapi.com',
     kip37: 'https://kip37-api.klaytnapi.com',
+    metadata: 'https://metadata-api.klaytnapi.com',
+    resource: 'http://resource-api.klaytnapi.com',
```

Comment:
```suggestion
    resource: 'https://resource-api.klaytnapi.com',
```
It should be `https`, right?